src/components/esn-popover/readme.md

```diff
@@ -0,0 +1,34 @@
+# esn-popover
+
+<!-- Auto Generated Below -->
+
+
+## Methods
+
+### `toggleShowState() => Promise<void>`
+
+A method to toggle the popover on/off.
+
+#### Returns
+
+Type: `Promise<void>`
+
+
+
+
```

Comment:
many blank lines

Reply:
It's an auto-generated file. We can leave it be.

Reply:
ok then